bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/users/UsersTableAccessor.java

```diff
@@ -22,19 +22,23 @@ public UsersTableAccessor(SQLiteConfig sqliteConfig) {
 	}
 
 	public User getUserById(int userId) throws SQLException {
-		String sql = "SELECT * FROM users WHERE userId = ? LIMIT 1";
-
 		Connection connection = getDatabaseConnection();
+		String sql = "SELECT id, first_name, last_name, email, password, created_at FROM users WHERE userId = ? LIMIT 1";
+
 		PreparedStatement sqlQuery = connection.prepareStatement(sql);
 		sqlQuery.setInt(1, userId);
 
 		ResultSet results = sqlQuery.executeQuery();
 
 		if (results.next()) {
+			String firstName = results.getString("first_name");
+			String lastName = results.getString("last_name");
 			String email = results.getString("email");
 			String password = results.getString("password");
-		}
+			Instant createdAt = results.getTimestamp("created_at").toInstant();
```

Comment:
We'll have to do the `Instant.parse(results.getString("created_at"))` here since SQLIte does not have the `TIMESTAMP` data type.